storm-core/src/jvm/org/apache/storm/Config.java

```diff
@@ -1872,6 +1872,13 @@
     @isString
     public static final String TOPOLOGY_BOLTS_TUPLE_TIMESTAMP_FIELD_NAME = "topology.bolts.tuple.timestamp.field.name";
 
+    /**
+     * Bolt-specific configuration for windowed bolts to specify whether late tuples should be emitted on a stream
+     * called _late, instead of being logged with INFO level.
+     */
+    @isBoolean
+    public static final String TOPOLOGY_BOLTS_EMIT_LATE_TUPLE = "topology.bolts.emit.late.tuple";
```

Comment:
Any reason for using a boolean and hardcoding the late tuple stream instead of letting users specify the late tuple stream (like in your earlier [patch](https://github.com/kosii/storm/commit/216c991da3c5b6c6cac1b25182b86507c3fb5e9e#diff-bf65d6d71c61b8fe2d118c4f7433de14R1880)) ? It may be better to let users specify the late tuple stream since it gives them more flexibility.